File: python_comparison/fib.py
import time

t0 = time.time()


def fib(n):
    if n == 1 or n == 0:
        return 1
    return fib(n - 1) + fib(n - 2)


N = 35
out = fib(N)

print(time.time() - t0, "seconds")

File: python_comparison/fib.ts
const t0 = performance.now()

const fib = (n: number): number => {
  if (n === 1 || n === 0) return 1
  return fib(n - 1) + fib(n - 2)
}

const N = 35

// eslint-disable-next-line @typescript-eslint/no-unused-vars
const out = fib(N)

console.log((performance.now() - t0) / 1e3, 'seconds')
